setup.py

```diff
@@ -59,7 +32,12 @@ def load_requirements(*requirements_paths):
         'License :: OSI Approved :: GNU Affero General Public License v3',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.5',
```

Comment:
"Programming Language :: Python :: 3" should also be included in order to appear on the Python 3 packages list on PyPI; more details at https://github.com/brettcannon/caniusepython3#how-do-you-tell-if-a-project-has-been-ported-to-python-3